<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forgetpassword</class>
 <widget class="QDialog" name="forgetpassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Forget Password</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Email">
        <property name="maximumSize">
         <size>
          <width>459</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="verify">
        <property name="maximumSize">
         <size>
          <width>459</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="verifyButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Verify</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetpassword_3">
        <property name="text">
         <string>Send Email</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>New password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pwd">
        <property name="maximumSize">
         <size>
          <width>437</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Confirm password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pwd2">
        <property name="maximumSize">
         <size>
          <width>437</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="resetpassword">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="HomePage">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
